src/OpenLoco/src/Objects/ObjectIndex.cpp

```diff
@@ -155,17 +154,27 @@ namespace OpenLoco::ObjectManager
         return currentState;
     }
 
+    static std::optional<bool> _customObjectsInIndex;
+
     // 0x00471712
-    static bool hasCustomObjectsInIndex()
+    bool hasCustomObjectsInIndex()
```

Comment:
I don't like the idea of a getter that has variable amount of work behind it. What was wrong with the old design?